tasks/copy_to_glacier/README.md

```diff
@@ -9,15 +9,13 @@ The `copy_to_glacier` module is meant to be deployed as a lambda function that t
 
 ## Exclude files by extension.
 
-You are able to specify a list of file types (extensions) that you'd like to exclude from the backup/copy_to_glacier functionality. This is done on a per-collection basis, configured in the `meta` variable of a Cumulus collection configuration:
+You are able to specify a list of file types (extensions) that you'd like to exclude from the backup/copy_to_glacier functionality. This is done on a per-collection basis, configured under `orca` key in the `OrcaExcludedFileRegEx` variable of a Cumulus collection configuration:
```

Comment:
Order seems off to me... Given the path "collection.orca.OrcaExcludedFileRegEx" the doc order is "orca" (2), "OrcaExcludedFileRegEx" (3), "collection" (1).

Maybe something like "configured in the Cumulus collection configuration under the `orca` key in the `OrcaExcludedFileRegEx` variable."?

Reply:
updated

Reply:
Missing the extra "meta" key.

Reply:
Extra " of a Cumulus collection configuration" at the end.

Reply:
removed